ykllvmwrap/src/ykllvmwrap.cc

```diff
@@ -21,15 +22,27 @@
 #include <err.h>
 #include <limits>
 #include <link.h>
+#include <mutex>
```

Comment:
Where does this library (no `.h`?) live?

Reply:
It's C++, so no .h suffix.

We can separate out the C and C++ headers if you like?

Reply:
Ah! I don't feel strongly.